src/nikhilMJun20/PalindromeNumber.java

```diff
@@ -0,0 +1,39 @@
+/*Verify given number is palindrome or not. 
+Input: number = 45654
+Output : Given number 45654 is palindrome.
+
+Input: number = 56789
+Output : Given number 56789 is not palindrome.
+*/
+
+package nikhilMJun20;
+
+import java.util.Scanner;
+
+public class PalindromeNumber {
+	void checkPalindromeNumber(int number) {
+		int revNumber = 0, originalNumber=number;
+		while(number>0) {
+			revNumber = 10*revNumber+(number%10);
+			number = number/10;
```

Comment:
To reduce this line, you can write operation on while condition like below
while((number = number/10)>0)

Reply:
Yes. This becomes even more compact.
Next time on I will use this style whenever possible.
Thank you, @KrishnaKTechnocredits !